### ISSUE TITLE
Audit JSON objects lack timestamps

### ISSUE THREAD

**Issue by mfischer-zd** [hashicorp/vault#471]
Audit JSON objects have no timestamps.  They should have them.

Instead of:

```
{"type":"request","auth":{"display_name":"root","policies":["root"],"metadata":null},"request":{"operation":"read","path":"sys/policy","data":null,"remote_address":"127.0.0.1"},"error":""}
```

There should be:

```
{"timestamp":"20150730T21:42:59.123", "type":"request","auth":{"display_name":"root","policies":["root"],"metadata":null},"request":{"operation":"read","path":"sys/policy","data":null,"remote_address":"127.0.0.1"},"error":""}
```

**Comment by armon** [comment]
Agreed!

**Comment by richard-hulm** [comment]
+1 This would be very useful!

**Comment by armon** [comment]
Fixed by #495

**Comment by mfischer-zd** [comment]
#495 still has issues, will update with another fix.
